source/Cosmos.System2_Plugs/System/IO/PathImpl.cs

```diff
@@ -242,7 +242,7 @@ public static string GetTempFileName()
 
         public static string GetTempPath()
         {
-            return @"\Temp";
+            return @"0:\Temp";
```

Comment:
Why is this change needed?

Reply:
Testing on Windows the value returned is indeed "C:/temp", it made some low level thing not work... Path or Directory I cannot remember correctly.

Reply:
The root drive should represent the system drive I think. "0:\\" may not be the root drive, so this needs to be fixed in the future.